src/main/java/com/sequenceiq/cloudbreak/service/stack/connector/aws/AwsConnector.java

```diff
@@ -347,8 +372,34 @@ private void updateInstanceMetadata(Stack stack, AmazonEC2Client amazonEC2Client
         }
     }
 
-    @Override
-    public CloudPlatform getCloudPlatform() {
-        return CloudPlatform.AWS;
+    private void disableAutoScalingGroups(Stack stack, AwsCredential awsCredential) {
```

Comment:
this method doesn't disable the autoscaling groups, it resumes all the scaling policies, because otherwise it can't be deleted (resumeAutoScalingPolicies)